staging/src/k8s.io/apiserver/pkg/storage/interfaces.go

```diff
@@ -40,10 +40,11 @@ type Versioner interface {
 	// from database.
 	UpdateObject(obj runtime.Object, resourceVersion uint64) error
 	// UpdateList sets the resource version into an API list object. Returns an error if the object
-	// cannot be updated correctly. May return nil if the requested object does not need metadata
-	// from database. continueValue is optional and indicates that more results are available if
-	// the client passes that value to the server in a subsequent call.
-	UpdateList(obj runtime.Object, resourceVersion uint64, continueValue string) error
+	// cannot be updated correctly. May return nil if the requested object does not need metadata from
+	// database. continueValue is optional and indicates that more results are available if the client
+	// passes that value to the server in a subsequent call. remainingItemCount indicates the number
+	// of remaining objects if the list is partial.
+	UpdateList(obj runtime.Object, resourceVersion uint64, continueValue string, remainingItemCount int64) error
```

Comment:
I guess a 0 value for remainingItemCount will result in the field being not set at all?

Reply:
Right. Added to the comment.